tools/mock-service-host/package.json

```diff
@@ -36,7 +36,7 @@
     "lodash": "^4.17.15",
     "mockjs": "^1.1.0",
     "oauth2-mock-server": "^3.1.0",
-    "oav": "~2.10.2",
```

Comment:
I see where you're going here, but do you want to [float the minor version?](https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept) 

Not that it's a huge risk given it should just be new features in minor releases, but still.

Reply:
Thanks @scbedd for taking look and catching.
I'm not using "^" intentionally, and I tend to use `float bug fix version` only for the sake of stability of mock server.

I know oav is planning on a new version, I'll update this PR later :)